src/Compute/Compute/Generated/Models/PSGalleryImageVersion.cs

```diff
@@ -49,6 +49,6 @@ public string ResourceGroupName
         public string Type { get; set; }
         public string Location { get; set; }
         public IDictionary<string, string> Tags { get; set; }
-
+        public string ReplicationMode { get; set; }
```

Comment:
looks like this is suppose part of GalleryImageVersion.PublishingProfile. 
So this line does not need to be added